src/_pytest/mark/__init__.py

```diff
@@ -96,7 +96,7 @@ def pytest_addoption(parser: Parser) -> None:
         dest="markexpr",
         default="",
         metavar="MARKEXPR",
-        help="only run tests matching given mark expression.\n"
+        help="Only run tests matching given mark expression.\n"
```

Comment:
```suggestion
        help="Only run tests matching given mark expression. "
```

Reply:
I understand we want to remove the `\n` and let the terminal/formatter handle new lines, correct? If not please disregard.

Reply:
Yes. I had decided to _not_ add any new ones, and leave the existing ones in, but it's a good idea to remove those too, there's not many.

Reply:
I see, thanks!